backend/Watchdog.Core/Watchdog.Core.BLL/Services/Queue/CollectorQueueConsumerService.cs

```diff
@@ -53,17 +53,15 @@ private async void Received(object sender, BasicDeliverEventArgs args)
             //TODO: Change to future value from IssueMessage
             int applicationId = 36;
 
-            using (var scope = _provider.CreateScope())
-            {
-                var userService = scope.ServiceProvider.GetRequiredService<IUserService>();
-                var userIds = await userService.GetUserUIdsByApplicationIdAsync(applicationId);
+            using var scope = _provider.CreateScope();
```

Comment:
Are you sure, that using syntax without brackets is better? I think it better looks with brackets

Reply:
In my opinion, the less nesting makes easier code reading.